modules/jolt_physics/objects/jolt_body_3d.cpp

```diff
@@ -690,6 +707,28 @@ void JoltBody3D::set_param(PhysicsServer3D::BodyParameter p_param, const Variant
 	}
 }
 
+void JoltBody3D::set_mass_properties(real_t p_mass, const Vector3 &p_center_of_mass, const Vector3 &p_inertia, const Vector3 &p_product_of_inertia) {
+	if (p_mass != mass) {
```

Comment:
The ifs that only assign a value in this function are redunant (3x), just assigning a value is cheaper in most cases.